util.sh

```diff
@@ -49,3 +49,10 @@ OCI_VERSION="${PODMAN_VERSION%.*}"
 FULL_IMAGE_NAME="${REPO}/${OCI_NAME}:${OCI_VERSION}"
 
 export FULL_IMAGE_NAME_ARCH="$FULL_IMAGE_NAME-${ARCH_TO_IMAGE_ARCH[$CPU_ARCH]}"
+
+BASE_FCOS_IMAGE="quay.io/fedora/fedora-coreos:41.20250215.3.0"
+export BASE_FCOS_IMAGE
+
+# koji tags prepend "f" to the release (rpm --eval)
+FEDORA_VERSION="f"$(podman run "$BASE_FCOS_IMAGE" rpm --eval '%{?fedora}')
+export FEDORA_VERSION
```

Comment:
This will keep the fedora / koji stuff in sync with coreos. PTAL

Reply:
ack, logic wise that seems good, however please add `--rm` to the podman command to not leak containers.
And also I don't think we should do that in util.sh because that is imported elsewhere and has no need for the fedora version. I suggest moving that into build.sh script.